### ISSUE TITLE
Add new contents page

### ISSUE THREAD

**Issue by prertik** [learning-lfe/learning-lfe.github.io#1]

**Comment by manuelcoppotelli** [comment]
Can you please specify more details?

**Comment by prertik** [comment]
@manuelcoppotelli If you have dabbled in the Erlang world, you might notice different languages like Elixir, LFE which needs Erlang VM other than Erlang. Elixir is more developed and maintained by lots of people. Many books are written about Elixir and has good documentation on its site. While LFE is maintained and overseen by Robert Virding only and there aren't good enough documentation and good enough resources to get started. The main purpose of learning-lfe is to solve this problem. So, in this issues, as you can see, there hasn't been anything written as the desire with which I started this kind of got overshadowed by other commitments and want to re-work on this project this 'Hacktober'. 

So, in conclusion, I want to create a new table of contents or a just contents page to plan the contents on this.
